.github/workflows/ci-admin.yml

```diff
@@ -0,0 +1,80 @@
+name: pmm-admin CI
+
+on:
+  push:
+    paths:
+      - "admin/**"
+    branches:
+      - main
+      - release-*
+    tags:
+      - v[0-9]+.[0-9]+.[0-9]+*
+  pull_request:
+    paths:
+      - "admin/**"
```

Comment:
We should  run CI for admin if API changed or something in the root of the project